Assets/MRTK/SDK/Features/UX/Scripts/Slate/HandInteractionPanZoom.cs

```diff
@@ -868,14 +868,15 @@ public void OnTouchCompleted(HandTrackingInputEventData eventData)
         /// </summary>
         public void OnPointerDown(MixedRealityPointerEventData eventData)
         {
+            bool isNear = eventData.Pointer as IMixedRealityNearPointer != null;
```

Comment:
Could this be simplified to:

```suggestion
            bool isNear = eventData.Pointer is IMixedRealityNearPointer;
```